dot/state/epoch.go

```diff
@@ -45,6 +52,13 @@ type EpochState struct {
 	blockState  *BlockState
 	epochLength uint64 // measured in slots
 	skipToEpoch uint64
+
+	nextEpochLock sync.RWMutex
+
+	// nextEpochData and nextConfigData is splited into
+	// epoch (uint64) and the block hash (common.Hash) that contains the digest
+	nextEpochData  map[uint64]map[common.Hash]types.NextEpochData
+	nextConfigData map[uint64]map[common.Hash]types.NextConfigData
```

Comment:
- Split the comments so it appears when hovering over it in IDEs
- They don't contain the `digest`, but a next-epoch-data or next-config-data

```suggestion
	// nextEpochData follows the format map[epoch]map[block hash]next epoch data
	nextEpochData  map[uint64]map[common.Hash]types.NextEpochData
	// nextConfigData follows the format map[epoch]map[block hash]next config data
	nextConfigData map[uint64]map[common.Hash]types.NextConfigData
```